test/integration/narrow/helpers/other-items-sizes.test.js

```diff
@@ -695,8 +695,8 @@ describe('Other Items Sizes Array Function', () => {
           {
             type: 'MIN_MAX',
             min: 1,
-            max: 9999,
-            error: 'Quantity must be between 1-9999'
+            max: 999999,
+            error: 'Quantity must be between 1-999999'
```

Comment:
![image](https://github.com/DEFRA/ffc-grants-slurry-web/assets/15157025/0f4afbef-c50a-4c61-bbc3-9abb70b2777b)
Inspection platform (this item) should be 1-9999 (look at safety equipment number 1